pkg/kubelet/kubelet.go

```diff
@@ -1040,6 +1086,15 @@ func (kl *Kubelet) pullImageAndRunContainer(pod *api.Pod, container *api.Contain
 		glog.Errorf("Error running pod %q container %q: %v", podFullName, container.Name, err)
 		return "", err
 	}
+	// Create a symbolic link to the Docker container log file using a name which captures the
+	// full pod name, the container name and the Docker container ID. Cluster level logging will
+	// capture these symbolic filenames which can be used for search terms in Elasticsearch or for
+	// labels for Cloud Logging.
+	containerLogFile := fmt.Sprintf("/var/lib/docker/containers/%s/%s-json.log", containerID, containerID)
```

Comment:
docker daemon has a flag --graph which allow to use a different path other than /var/lib/docker as root of the Docker runtime. Hard-coded the path here might break some users.